### ISSUE TITLE
V5 Modals with size specified are displayed incorrectly

### ISSUE THREAD

**Issue by Noryoko** [chanan/BlazorStrap#481]
The size class is applied to both the modal div and the modal-dialog div causing the modal to be displayed on the left side of the page rather than horizontally centered. I think this could be resolved by just applying it to the modal-dialog div.

```
<BSModal @ref="modalX" Size="Size.Large">
    <Header>Some title here</Header>
    <Content>Some content here</Content>
    <Footer>
        <BSButton Color="BSColor.Primary" @onclick="modalX.HideAsync">Close</BSButton>
    </Footer>
</BSModal>
```

**Comment by jbomhold3** [comment]
Sorry about the delay will address this today

**Comment by jbomhold3** [comment]
Fixed in .103 no workaround was available

**Comment by Noryoko** [comment]
Issue is fixed with 5.0.103. Thank you very much!
